modules/ROOT/pages/create-api-specification-studio.adoc

```diff
@@ -0,0 +1,41 @@
+= Create an API Project in Studio
+
+Anypoint Studio 7.7 and later enables you to create a RAML or OAS 2.0 and 3.0 API specification, or an API fragment in Studio and automatically synchronize it with Design Center.
+
+It is not possible to create a flow out of an OAS 3.0 API specification.
+
+[NOTE]
+Studio uses the EGit plugin only for the VCS feature of editing API specifications offline. Studio does not support the EGit plugin for Mule application projects that you might track using your own VCS.
```

Comment:
Similarly, is the the proper place to describe editing specifications, when we have only introduced creating them?

Reply:
Looking ahead at the whole topic, I see two subtopics that are not mentioned in the topic set intro.

This makes me wonder if this should be THREE subtasks to creating a PROJECT:
== Create API SPEC or FRAGMENT
== Create branches.
== Merge and push(or synchronize: choose one term) PROJECT

However, two of these are not tasks, because they have no steps but rather just links. So maybe this should really be just one task with no subtopics:

To create an API project in Studio:
1. Log in to Anypoint Platform.
2. In the task bar....
3. Select the type:
     * spec
     * frag
4. Type name.
5. Select finish.
6. Optionally, create branches to...:
    <links>
7. Merge and sync/push
    <link>

Consider enhancing the intro to describe that creating a project involves deciding to create a spec or fragment and that you can then edit (EGit) or track development/maintain sync using VCS (but these really aren't part of the main topic, which is currently titled CREATE).

Reply:
OK...just edits to existing text from here on...: